docs/docs/install/configuration.md

```diff
@@ -0,0 +1,34 @@
+# Configuring Mathesar 
+
+The table below shows all available environment variables supported by Mathesar. See the specific installation guides for the applicable environment variables and instructions on how to set them.
+
+> Variables marked with *Internal* should only be changed if you know what you are 
+> doing.
+
+
+### Backend Configuration
+| Name                   | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | Defaults                     |
+|------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------|
+| SECRET_KEY             | The Django secret key consists of random 50 characters strings is used to provide cryptographic signing. This key is mostly used to sign session cookies. Refer [Django docs](https://docs.djangoproject.com/en/3.2/ref/settings/#std:setting-SECRET_KEY) for more details. You can either generate a random 50 character string by yourself or you can generate a secret key using [this tool](https://djecrety.ir/)                                                                                                                               | *Required                    |
```

Comment:
> *Required

It confuses me on why the `Defaults` column contains `*Required`. I don't think we should place it there. Perhaps we can place it in the `Name` column like:

> **SECRET KEY** *(Required)*

and the Defaults column can contain `-` or *(empty)* for empty values.